clang/test/Driver/sycl-bfloat16-lib-win.cpp

```diff
@@ -3,52 +3,68 @@
 ///
 
 // REQUIRES: windows
-// REQUIRES: opencl-aot, ocloc, cpu, gpu
 // UNSUPPORTED: cuda
 
 /// ###########################################################################
-/// test that no bfloat16 libraries are added in JIT mode
+/// Test that no bfloat16 libraries are added in JIT mode.
 // RUN: %clangxx -fsycl %s --sysroot=%S/Inputs/SYCL -### 2>&1 \
 // RUN:   | FileCheck %s -check-prefix=BFLOAT16
 
-// test that fallback bfloat16 libraries are added in JIT mode with generic target
-// RUN: %clangxx -fsycl -fsycl-targets=spir64 %s -### 2>&1 \
+// Test that fallback bfloat16 libraries are added in JIT mode with \
```

Comment:
The escape at the end of comment lines is not necessary

Reply:
Right. I'll remove them.